### ISSUE TITLE
[Issue]: Inconsistent metadata artistIds tag separator

### ISSUE THREAD

**Issue by rasmuslos** [jellyfin/jellyfin#10301]
### Please describe your bug

The Musicbrainz-ArtistIds metadata field has different separators, depending on the type of the file. When parsing a Flac file `/` is used, while mp3 files use `;`. I have no good explanation for why this is happening, as both ID3 2.4 and Vorbis comments support multiple tag values ... Some plugins like the listenbrainz integration assume the separator always to be `;`, so they don't work as expected

### Jellyfin Version

Other

### if other:

unstable

### Environment

```markdown
- OS:
- Linux Kernel:
- Virtualization: Docker
- Clients: Web
- Browser: Safari
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
I'm not sure how we can cater to this without keeping specific exclusion lists for artists (which is a bad workaround imho).
If you are tagging with Picard: It usually does not write multi-tag if you do not instruct it to, so it uses whatever MusicBrains uses.

**Comment by rasmuslos** [comment]
I wrote both the mp3 and flac tags using Picard, so I _assume_ they are pretty similar. I also tried some things like manually encoding data using ffmpeg and changing some settings in Picard, but nothing changed this.
While an exclusion list would be required for the artists names the main think that I causing problems, at least for me, are the artist musicbrainz ids (`MUSICBRAINZ_ARTISTID`). They only consist of letters, numbers and dashes (and the separator). Is it possible to streamline them?
